### PR TITLE
Clarify "Authentication is required to perform this action" demands user password rather than root password

### PR DESCRIPTION
**Pull request by hickford**
Fixes https://github.com/mate-desktop/mate-polkit/issues/13

I don't know C, so please correct my guess before you merge.

### CODE REVIEW

**Comment by flexiondotorg** [comment]
But this authentication is not using sudo, it uses PolicyKit.

**Comment by hickford** [comment]
@flexiondotorg You're right. How about?

> Authentication is required to perform this action. **Your password**:

File: src/polkitmateauthenticationdialog.c [hunk 1]
```diff
@@ -637,6 +637,7 @@ polkit_mate_authentication_dialog_constructed (GObject *object)
   GtkWidget *action_area;
   gboolean have_user_combobox;
   gchar *s;
+  gchar *password_label;
```
Comment (hickford) [inline src/polkitmateauthenticationdialog.c]: Is `gchar` the correct type here?

**Comment by raveit65** [comment]
NO action since several years, closing.
Blame  me if this was wrong.

**Pull request closed without merging**